sensors/src/main/scala/com/lynbrookrobotics/potassium/sensors/imu/DigitalGyro.scala

```diff
@@ -48,11 +47,32 @@ abstract class DigitalGyro(tickPeriod: Time) {
     if (calibrating) {
       calibrationVelocities.enqueue(retrieveVelocity)
 
-      if (calibrationVelocities.size > 200) calibrationVelocities.dequeue()
+      if (calibrationVelocities.size > 1000) calibrationVelocities.dequeue()
 
       Value3D(DegreesPerSecond(0), DegreesPerSecond(0), DegreesPerSecond(0))
     } else {
       retrieveVelocity - currentDrift
     }
   }
+
+  def checkCalibrationValid(maxDriftDeviation: Value3D[AngularVelocity]): Boolean = {
+    val chunks: Seq[Seq[Value3D[AngularVelocity]]] = {
+      val part1 = calibrationVelocities.splitAt(calibrationVelocities.size / 5)
+      val part2 = part1._2.splitAt(calibrationVelocities.size / 5)
+      val part3 = part2._2.splitAt(calibrationVelocities.size / 5)
+      val part4 = part3._2.splitAt(calibrationVelocities.size / 5)
+      Seq(part1._1.toList, part2._1.toList, part3._1.toList, part4._1.toList, part4._2.toList)
```

Comment:
These `toList`s are a performance burden since they forcibly convert the sequences into linked lists.